packages/account/src/Styles/account.scss

```diff
@@ -961,6 +958,9 @@ $MIN_HEIGHT_FLOATING: calc(
             flex: 1;
         }
         &-title {
+            font-size: inherit;
+            font-weight: inherit;
+
```

Comment:
🤔 
the aim of using <Text> component is to remove all CSS related to text 
is it possible to handle this on the code?